.github/workflows/validateAllAddons.yml

```diff
@@ -32,6 +32,29 @@ jobs:
       run: |
         python -m pip install --upgrade wheel
         pip install -r validation/requirements.txt
+    - name: Install addon download validation dependencies
```

Comment:
this should probably be moved further up - e.g. below creating validation errors file